tests/networks/test_lazy_onnx_import.py

```
@@ -0,0 +1,63 @@
+    """Regression tests for #8455.
+
+    Importing MONAI must not eagerly import onnx / onnxruntime. These were
+    previously imported at module scope via ``optional_import`` in
+    ``monai/networks/utils.py`` and ``monai/bundle/scripts.py``, so a
+    broken/hanging onnx (e.g. onnx 1.18 on Windows) would take down
+    ``import monai`` entirely.
+    """
+
+    def _assert_not_eagerly_imported(self, import_stmt: str) -> None:
+        code = _PROBE.format(import_stmt=import_stmt)
+        result = subprocess.run([sys.executable, "-c", code], capture_output=True, text=True)
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)